### PR TITLE
Reader - tweak styles on following settings popover

### PR DESCRIPTION
**Pull request by blackjackkent**
#### Changes proposed in this Pull Request

* This diff updates styling on the per-blog notification settings popup on `/following/manage`, as discussed in https://github.com/Automattic/wp-calypso/issues/56117

#### Testing instructions

* Check out this diff locally and navigate to http://calypso.localhost:3000/following/manage
* Verify that updated styles have been applied

Before:
image

After:
image

Related to #56117

### CODE REVIEW

**Comment by kwight** [comment]
Looks good – there's one part, the triangle, that is still the old colour though:
Screen Shot 2021-09-20 at 11 01 27 AM

cc/ @saygunnyc Cool, it does look better being more consistent with eg. domain popovers (FYI @bluefuton in case there are any concerns we can't think of).

**Comment by kwight** [comment]
Oh, maybe the border thickness of the triangle is too thick too. 🤔

**Comment by blackjackkent** [comment]
Pushed a fix updating the background of the arrow portion - good eye!

I *think* the border thickness on the triangle might be an optical illusion, but I'm also a little confused about how that particular knot of CSS works; maybe @sixhours can weigh in if time permits?

**Comment by kwight** [comment]
> I think the border thickness on the triangle might be an optical illusion, but I'm also a little confused about how that particular knot of CSS works

Oh, you may be right (it's the same in prod now). I think borders are the `stroke` for `path`s.

Looks good to me; let's give until the end of day tomorrow before merging so that others can chime in if needed.

**Review by kwight — APPROVED** [review]

**Comment by mreishus** [comment]
Here's how the triangle works. Some tricky CSS is used to turn regular, thick, box borders into a triangle. Then to make the outline, a second triangle is placed on top of the first one with absolute positioning, offset by two pixels:


https://user-images.githubusercontent.com/937354/134074724-509775d2-de79-460c-877a-92bc6860901f.mp4


https://user-images.githubusercontent.com/937354/134074753-6968e084-6302-4815-83e6-67ed88f355d9.mp4

Changing the `bottom: 2px` rule to 1px on the ::before makes it thinner. I don't have a good eye for design so I will make no judgements about which thickness is correct :)

**Comment by saygunnyc** [comment]
Great eye with the stroke weight and the background color @kwight! The stroke weight should be the same on the whole component. All looks good other than those 2 mentioned by you.

It seems like the stroke weight was also off over the existing component.

**Comment by kwight** [comment]
> Changing the bottom: 2px rule to 1px on the ::before makes it thinner. I don't have a good eye for design so I will make no judgements about which thickness is correct :)

Yeahhhhhh... `2px` looks too thick, while `1px` looks a little thin. ¯\_(ツ)_/¯

Screen Shot 2021-09-21 at 9 16 32 AM

Final decision to @saygunnyc : keep 2px, or change them all to 1px? @blackjackkent regardless, let's make sure this is merged by EOD tomorrow at the latest to make the sprint.

**Comment by saygunnyc** [comment]
> Yeahhhhhh... 2px looks too thick, while 1px looks a little thin. ¯_(ツ)_/¯
Final decision to @saygunnyc : keep 2px, or change them all to 1px?

After double-checking a few more instances – esp. overflow menus over Calypso, I can see that we've been using 2px weight on diagonal lines as they usually have this bad habit of acting exactly the same way you described @kwight 😄 

For the sake of this PR and consistency, let's **keep the 2px border weight**.

**Comment by kwight** [comment]
> For the sake of this PR and consistency, let's keep the 2px border weight

Awesome – merge away @blackjackkent 👍

**Pull request merged**
